### PR TITLE
update pybglibpy version to 4.8.2

### PR DESCRIPTION
**Pull request by anilbey**

### CODE REVIEW

**Review by wvangeit — DISMISSED** [review]

**Comment by anilbey** [comment]
I can't restart the failed jobs. Last time I was able to do that. How should I proceed @matz-e ?

**Comment by anilbey** [comment]
And the logs have a lot of noise. The reason of failure is not shown. In the end the output exceeds the limit.

Maybe those condition lines can be grouped (or even removed). E.g. 25 conditions per line.

```
  condition(1190)
  condition(1191)
  condition(1192)
  condition(1193)
  condition(1194)
  condition(1195)
  condition(1196)
  condition(1197)
  condition(1198)
  condition(1199)
  condition(12)
  condition(120)
  condition(1200)
  condition(1201)
  condition(1202)
  condition(1203)
  condition(1204)
  condition(1205)
  condition(1206)
  condition(1207)
  condition(1208)
  condition(1209)
  condition(121)
  condition(1210)
  condition(1211)
  condition(1212)
  condition(1213)
  condition(1214)
  condition(1215)
  condition(1216)
  condition(1217)
  condition(1218)
  condition(1219)
  condition(122)
  condition(1220)
  condition(1221)
  condition(1222)
  condition(1223)
  condition(1224)
  condition(1225)
  condition(1226)
  condition(1227)
  condition(1228)
  condition(1229)
  condition(123)
  condition(1230)
  condition(1231)
  condition(1232)
  condition(1233)
  condition(1234)
  condition(1235)

...

  condition_requirement(5165,"version_satisfies","py-jupyter-client","6.1.13:")
  condition_requirement(5166,"node","py-jupyter-client")
  condition_requirement(5166,"version_satisfies","py-jupyter-client","5:")
  condition_requirement(5167,"node","py-jupyter-client")
  condition_requirement(5168,"node","py-j
[33;1mJob's log exceeded limit of 4194304 bytes.
Job execution will continue but no more output will be collected.[0;m
```

File: bluebrain/repo-bluebrain/packages/py-bglibpy/package.py [hunk 1]
```diff
@@ -24,6 +24,7 @@ class PyBglibpy(PythonPackage):
     depends_on('py-cachetools', type='run')
     depends_on('py-bluepy@2.4.2:2.999', type='run')
     depends_on('py-bluepy-configfile@0.1.18:', type='run')
+    depends_on('py-pandas@1.4.0:', type='run')
```
Comment (matz-e) [inline bluebrain/repo-bluebrain/packages/py-bglibpy/package.py]: We don't have pandas 1.4.0 in Spack yet, that's why stuff is failing :)

Comment (anilbey) [reply bluebrain/repo-bluebrain/packages/py-bglibpy/package.py]: Thanks! It's updated now :)

**Comment by anilbey** [comment]
Hi @matz-e the gitlab tests seem to be working now.
There's this error in linux tests/ unittest (3.6, original) 
Do you know what might be causing it?


```python
  def check_output(ni, na):
        with capsys.disabled():
            output = extensions("python")
            packages = extensions("-s", "packages", "python")
            installed = extensions("-s", "installed", "python")
            activated = extensions("-s", "activated", "python")
        assert "==> python@2.7.11" in output
>       assert "==> 3 extensions" in output
E       AssertionError: assert '==> 3 extensions' in '==> python@2.7.11%gcc@4.5.0 arch=test-debian6-core2/e6jikt4\n==> 2 extensions:\npy-extension1\npy-extension2\n==> 2 i...debian6-core2 / gcc@4.5.0 -------------------------------\npy-extension1@2.0\npy-extension2@1.0\n==> None activated.\n'
```

**Comment by matz-e** [comment]
This is a known issue that _just_ got fixed. I'll merge anyways.

**Comment by anilbey** [comment]
Thanks!

**Pull request merged**